client/cli/src/libra_client.rs

```diff
@@ -285,24 +282,26 @@ impl LibraClient {
             TrustedStateChange::Epoch {
                 new_state,
                 latest_epoch_change_li,
-                latest_validator_set,
-                ..
             } => {
                 info!(
                     "Verified epoch change to epoch: {}, validator set: [{}]",
                     latest_epoch_change_li.ledger_info().epoch(),
-                    latest_validator_set
+                    latest_epoch_change_li
+                        .ledger_info()
+                        .next_validator_set()
+                        .expect("no validator set in epoch change ledger info"),
```

Comment:
this is exactly why e.g. `TrustedStateChange::Epoch` contained a `&ValidatorSet`, so you don't have to double check that the ledger info is an epoch change ledger info. was there a reason to remove it?

Reply:
don't have strong opinion, it's removed in previous PR, just tried to minimize the struct